### PR TITLE
Merge makefile 

### PR DESCRIPTION
**Pull request by Pandabear1125**

### CODE REVIEW

**Comment by Pandabear1125** [comment]
Makefile is functional (and easy to use), ready to be used :)

File: libraries/Adafruit_ICM20X/.DS_Store
Comment (StudentKyle) [inline libraries/Adafruit_ICM20X/.DS_Store:1]: .DS_Store is a MacOS file structure thing that isn't needed. We should probably remove it, likely by adding it to the gitignore (there are .DS_Store files elsewhere as well)

**Review by StudentKyle — CHANGES_REQUESTED** [review]
Add .DS_Store to gitignore

**Review by Rishi-H — APPROVED** [review]
Looks good to me. Thanks for spending the time to do this.

**Comment by Pandabear1125** [comment]
Keeping branch open in case makefile requires revision.

**Pull request merged**